### ISSUE TITLE
Mojave (fresh install), SIP disabled, Thunderbolt Display 1920 x 1080 HiDPI not working

### ISSUE THREAD

**Issue by story1234** [usr-sse2/RDM#28]
I have no idea how you guys do this.  I have 2 MacBooks Pro 2013 (13 and 15") BOTH doesn't work.

1. Fresh install Mojave
2. Disable SIP
3. Open Safari and download this fork of RDM
4. Add 1920 x 1080 with HiDPI check
5. Reboot
6. RDM shows the lightning icon with the resolution 1920 x 1080
7. Nothing happens
8. Repeat above with 3840x2160
9. Same and post here.

1280x800 works fine.

**Comment by jnooree** [comment]
It is unclear to me what is actually happening. Is it

1) 1920 * 1080 resolution with lightning icon does not show up, or
2) selecting 1920 * 1080 resolution with lightning icon does nothing

Please answer this so that I can help you solve your problem.

**Comment by story1234** [comment]
Sorry, I didn't add a step where I select.
It's 2. selecting 1920 * 1080 resolution with lightning icon does nothing

Thanks.

**Comment by jnooree** [comment]
First, which version is in use? The latest release (`2.3.3`) or the `HEAD` version (the latest commit)?
Second, I recommend you to remove the plist file you've made under the `/System/Library/` directory before any other operations. Make sure **not to remove all the plists**, because most of the plist files are bundled with macOS.

**Comment by story1234** [comment]
I am using 2.3.3.  I don't see any plist files being made.  As this is a fresh install.

**Comment by jnooree** [comment]
Sorry for confusion. The full path is `/System/Library/Displays/Contents/Resources/Overrides`, and you have to find the directory which was modified recently.

**Comment by story1234** [comment]
Yes, nothing there.  It's all files from 2019. Maybe there plist was never created even though it shows the icon?

**Comment by story1234** [comment]
https://imgur.com/q6qK6BH

**Comment by jnooree** [comment]
That's one possibility. Could you paste the result of the below command? (At the `.../Overrides` folder)
`sudo find -type f -atime -10`
The command just finds all files which was modified within 10 days.

**Comment by story1234** [comment]
https://imgur.com/Bv1ZwVt

**Comment by jnooree** [comment]
Oh yours is not a GNU find... How about `sudo find . -type f -newermt 20201201`?

**Comment by story1234** [comment]
./DisplayVendorID-610/DisplayProductID-9227

**Comment by story1234** [comment]
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>DisplayProductID</key>
	<integer>37415</integer>
	<key>DisplayProductName</key>
	<string>Thunderbolt Display2</string>
	<key>DisplayVendorID</key>
	<integer>1552</integer>
	<key>scale-resolutions</key>
	<array>
		<data>
		AAAHgAAABDgLux0YAAAAAQ==
		</data>
		<data>
		AAAPAAAACHALux0YAAAAAQ==
		</data>
	</array>
</dict>
</plist>

**Comment by story1234** [comment]
I just tried on another MacBook Pro 13 (2013) with Mojave that's connected to a TV (1080p) by HDMI.  It showed 1080p with Lightning symbol but clicking on it does nothing.  Am I the only one that couldn't get it work on Mojave?  This is getting weird.
